src/pages/Admin/Venue/VenueWizard/VenueWizard.tsx

```diff
@@ -3,21 +3,18 @@ import React, { useReducer } from "react";
 import { useVenueId } from "hooks/useVenueId";
 
 import VenueWizardCreate from "./Create";
-// Components
```

Comment:
Removed import comments as to not trip over the automated import sorting algorithm.